subgraphs/balancer-v2/src/common/getters.ts

```diff
@@ -35,10 +35,7 @@ export function getOrCreateDex(): DexAmmProtocol {
     protocol.subgraphVersion = "1.2.0";
     protocol.methodologyVersion = "1.2.0";
```

Comment:
methodology version should align with the version in the README.

we can bump the schema version to 1.2.1 now and update the schema (just changes in comments)